<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LessonDialog</class>
 <widget class="QDialog" name="LessonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LessonDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Teacher</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Choose teacher</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mTeacher"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Subject</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Choose subject</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mSubject"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Class</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Choose class</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mClasses"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hours per week</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="mHoursPerWeek"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="Ok">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Ok</sender>
   <signal>clicked()</signal>
   <receiver>LessonDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
